pkg/scheduler/plugins/conformance/conformance.go

```diff
@@ -21,9 +21,24 @@ import (
 	"k8s.io/kubernetes/pkg/apis/scheduling"
 
 	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/api"
+	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/conf"
 	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/framework"
 )
 
+const name = "conformance"
+
+func init() {
```

Comment:
please do not modify plugins one by one, that's ok to set default directly after we load config.

Reply:
Sorry, I do not catch it. How to set default directly?